src/modules/m_override.cpp

```diff
@@ -27,10 +27,23 @@
 #include "inspircd.h"
 #include "modules/invite.h"
 
+class Override : public SimpleUserModeHandler
+{
+ public:
+	Override(Module* Creator) : SimpleUserModeHandler(Creator, "Override", 'O')
+	{
+		oper = true;
+		if (ServerInstance->Config->ConfValue("override")->getBool("enableumode"))
```

Comment:
I think this is inverted? Surely you should only call `DisableAutoRegister()` if the mode is disabled?